scripts/cat/cats.py

```diff
@@ -3106,13 +3088,12 @@ def mediate_relationship(mediator, cat1, cat2, allow_romantic, sabotage=False):
 
     @staticmethod
     def effect_relation(current_value, effect):
-        if effect < 0 and abs(effect) >= current_value:
-            return 0
-
-        if effect > 0 and current_value + effect >= 100:
-            return 100
-
-        return current_value + effect
+        new_value = current_value + effect
+        if new_value < 0:
+            new_value = 0
+        elif new_value > 100:
+            new_value = 100
+        return new_value
```

Comment:
It can indeed be compressed to 1 line, tenary operation is also an option.

But I think the if-else is more readable for a human, which is why I did it that way. If min-max is prefered I'll change it to that

Reply:
As an alternative, I could suggest adding a helper function `clamp` in any appropriate location and using that instead; you could describe the purpose of the clamp-function with a docstring making it easy for people to comprehend if they're unsure what "clamp" means in this context and provide us with a neat utility to use down the line.

Reply:
I have now implemented a clamp in scripts->utility.py and call that. Therefore I will resolve this conversation.